### PR TITLE
Optimize desktop view

### PR DESCRIPTION
**Pull request by awang-karisma**
Show 3 and 4 columns of card in lg and xl breakpoint respectively, so desktop user will have their screen filled instead of empty space.

### CODE REVIEW

**Comment by angeloanan** [comment]
cc @resir014 AFAIK you're doing some redesign on this. Those this line up with your changes?

**Comment by angeloanan** [comment]
#### 📝 Changed routes:


| [**/**](https://kominfo-2fl3ssscx-angewo.vercel.app/) |
|:-:|
|<a href="https://kominfo-2fl3ssscx-angewo.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/z84KQUGp04ngKEi7ZUiFDi68Y0KyfHAj8rsahIpEHjFyyWJa5yrTEFw1v6erlz62GMcqqlcQqwJypTRNLPFYXg.png">(view full size)</a> |







Commit f79956d53bf24734cfaae33dc81e7771f92f3ef5 (https://kominfo-2fl3ssscx-angewo.vercel.app).

**Comment by awang-karisma** [comment]
Ah sorry I didn't know someone already doing the redesign, it's my first public PR so bear with me 🤣

File: src/pages/index.tsx [hunk 1]
```diff
@@ -73,13 +73,13 @@ const IndexPage = ({ PSEData: data, blockData, trustPositifData }: IndexPageProp
       <div className='flex flex-col bg-brand-bg-light'>
         <SiteHeader />
         <PageContent>
-          <div className='mx-auto w-full max-w-screen-sm'>
+          <div className='mx-auto w-full'>
```
Comment (resir014) [inline src/pages/index.tsx:76]:
```suggestion
          <div className='mx-auto w-fullmax-w-screen-xl'>
```

Idem.

**Review by resir014 — CHANGES_REQUESTED** [review]
Some changes needed.

File: src/components/SiteHeader.tsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ import { KominfodLogo } from './KominfodLogo'
 export function SiteHeader() {
   return (
     <header className='flex h-[60px] w-full items-center bg-brand-bg px-4'>
-      <div className='mx-auto flex w-full max-w-screen-sm items-center justify-between'>
+      <div className='mx-auto flex w-full items-center justify-between'>
```
Comment (resir014) [inline src/components/SiteHeader.tsx:9]:
```suggestion
      <div className='mx-auto flex w-full max-w-screen-sm items-center justify-between'>
```

This is still too big on desktop. Please at least provide a max-width.

image

**Review by resir014 — CHANGES_REQUESTED** [review]

File: src/pages/index.tsx (hunk 1 above)
Comment (angeloanan) [reply src/pages/index.tsx:76]: Chiming in, old version of the website uses `max-w-screen-xl`
https://github.com/angeloanan/kominfo-d-/blob/5a958fbbd4aacb1fc41c78d88b5cb42034b04a61/src/pages/index.tsx#L71

**Comment by angeloanan** [comment]
Merging on build success

**Pull request merged**